quantumdb-postgresql/src/main/java/io/quantumdb/core/planner/CatalogLoader.java

```diff
@@ -316,7 +316,7 @@ private static void addIndexes(Connection connection, Catalog catalog, String ta
 				parser.present("CONCURRENTLY");
 				String indexName = parser.consume();
 				indexName = removeOuterQuotes(indexName);
-				if (indexName.startsWith("pk_")) {
+				if (indexName.endsWith("_pkey")) {
```

Comment:
Primary key indexes end with "_pkey". Before this change, all indexes made by primary key constraints were duplicated.